### DISCUSSION TITLE
can not find uuid.webview.dd.ss.cn 's server IP address

### DISCUSSION THREAD

**Discussion by yy0827-mengjiang** [eclipse-theia/theia#12344] [Q&A]
Hi,when I try to open a file in browser, it will show me “can not find uuid.webview.dd.ss.cn 's server IP address。”，if my IDE's address is https://dd.ss.cn/ide/xxxx(ingressId)/#/home/project.Can I modify this address so that the network can access it？

**Comment by VyacheslavKozloff** [comment]
You need to add the environment variable THEIA_WEBVIEW_EXTERNAL_ENDPOINT="{{hostname}}".
By default, theia (browser) starts with the value THEIA_WEBVIEW_EXTERNAL_ENDPOINT="{{uuid}}-webview-{{hostname}}".
Or when building the image, add the line ENV THEIA_WEBVIEW_EXTERNAL_ENDPOINT={{hostname}} to the Dockerfile
